shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/DMLStatement.g4

```diff
@@ -198,15 +198,23 @@ usingClause
     ;
 
 mergeUpdateClause
-    : WHEN MATCHED THEN UPDATE SET mergeAssignment (COMMA_ mergeAssignment)* whereClause? (DELETE whereClause)?
+    : WHEN MATCHED THEN UPDATE mergeSetAssignmentsClause whereClause? deleteWhereClause?
```

Comment:
Yes @wgy8283335, `WHEN MATCHED THEN UPDATE SET`  is better. I put `SET` separately in the `mergeSetAssignmentsClause`  because of its rule name only. Since the rule name specifies `SetAssignments` in its name, I just put keyword `SET` inside that rule. 

I'll change that one as: 
```
mergeUpdateClause
    : WHEN MATCHED THEN UPDATE SET mergeSetAssignmentsClause whereClause? deleteWhereClause?
    ;

mergeSetAssignmentsClause 
    : mergeAssignment (COMMA_ mergeAssignment)*
```

Reply:
Hi @wgy8283335, I've updated `mergeSetAssignmentsClause`.